USE-CASES/MedicalDevices.md

```diff
@@ -0,0 +1,110 @@
+
+In recent years, researchers have made progress developing PCLC devices for mechanical ventilation, anesthetic delivery applications, and so on. Despite these promises and potential benefits, there has been limited success in the translation of PCLC devices from [bench to bedside](https://today.duke.edu/2014/07/benchbedside). A key challenge to bringing PCLC devices to a level required for a clinical trials in humans is risk management to ensure device reliability and safety.
+
+The United States Food and Drug Administration (FDA) classifies new hazards that might be introduced by PCLC devices into three categories. Besides clinical factors (e.g. sensor validity and reliability, inter- and intra-patient physiological variability) and usability/human factors (e.g. loss of situational awareness, errors, and lapses in operation), there are also engineering challenges including robustness, availability, and integration issues.
+
+#### Variants:
+
+US military developed ONR SBIR (Automated Critical Care System Prototype), and found those issues.
+
+- No plug and play, i.e. cannot swap O2 Sat with another manufacturer.
+- No standardization of data outputs for devices to interoperate.
+- Must have the exact make/model to replace a faulty device or system will not work.
+
+### Security Considerations:
+
+Security considerations for interconnected and dynamically composable medical systems are critical not only because laws such as [HIPAA](https://www.hhs.gov/hipaa/index.html) mandate it, but also because security attacks can have serious safety consequences for patients. The systems need to support automatic verification that the system components are being used as intended in the clinical context, that the components are authentic and authorized for use in that environment, that they have been approved by the hospital’s biomedical engineering staff and that they meet regulatory safety and effectiveness requirements.
+
+For security and safety reasons, [ICE (F2761-09)](#ICE) medical devices never interact directly each other. All interaction is coordinated and controlled via the applications.
+
+While transport-level security such as TLS provides reasonable protection against external attackers, they do not provide mechanisms for granular access control for data streams happening within the same protected link. Transport-level security is also not sufficiently flexible to balance between security and performance. Another issue with widely used transport-level security solutions is the lack of support for multicast.
+
+### Privacy Considerations:
+
+<Describe any issues related to privacy; if there are none, say "none" and justify>
+
+### Gaps:
+
+Multicast support. It has proven useful for efficient and scalable discovery and information exchange in industrial systems.
+
+### Existing standards:
+
```

Comment:
There are  https://www.pchalliance.org/ and https://www.hl7.org - It would be nice to check the status and technical choices.